README.md

```diff
@@ -76,6 +76,8 @@ information on how to submit your changes to this repository.
 
 ## Sample view
 
+See ./examples for docker-compose file with Grafana if you'd like to quickly get a test environment up and running.
```

Comment:
This is the article I used to create it. It was extremely helpful in learning:

https://finestructure.co/blog/2016/5/16/monitoring-with-prometheus-grafana-docker-part-1

Here's the official guide(Example above I believe is more helpful):

https://docs.docker.com/compose/